pkgs/libresoc/openpower-isa.nix

```diff
@@ -34,6 +34,11 @@ with python39Packages;
         --replace 'os.environ.get(cmd.upper(), default)' 'default'
     '';
 
+    patches = [
+      # setup.py uses @git syntax for version specifiers, prevent setuptools from attempting to clone
+      ./use-vendored-git-dependencies.patch
```

Comment:
@jleightcap move adding this patch file into this commit

Reply:
pip will try to clone git repos, but this patch makes it so we can use the repos that have already been vendored in by nix